manifests/params.pp

```diff
@@ -30,7 +30,7 @@
       case $facts['os']['release']['major'] {
         '7': {
           $additional_packages = ['easy-rsa']
```

Comment:
Should this now also be `$additional_packages = ['easy-rsa','openvpn-auth-ldap']` for EL7 as well as EL6?
(If these parameters are now the same on EL6 and EL7, they can also be moved out of this case statement).

Reply:
Did EL7 packages become available some time *after* https://github.com/voxpupuli/puppet-openvpn/pull/215 ?

Reply:
It looks like the openvpn-auth-ldap package is now available in the repo for EL7. Will amend the PR.